sdk/confidentialledger/Azure.Security.ConfidentialLedger/src/ConfidentialLedgerCertificateClient.cs

```diff
@@ -4,33 +4,32 @@
-using Azure;
 using Azure.Core;
 using Azure.Core.Pipeline;
 
-namespace Azure.Security.ConfidentialLedger
+namespace Azure.Security.ConfidentialLedger.Certificate
 {
-    [CodeGenSuppress("ConfidentialLedgerIdentityServiceClient", typeof(Uri), typeof(TokenCredential), typeof(ConfidentialLedgerClientOptions))]
-    [CodeGenSuppress("ConfidentialLedgerIdentityServiceClient", typeof(Uri), typeof(TokenCredential))]
-    public partial class ConfidentialLedgerIdentityServiceClient
+    [CodeGenSuppress("ConfidentialLedgerCertificateClient", typeof(Uri), typeof(TokenCredential), typeof(ConfidentialLedgerClientOptions))]
+    [CodeGenSuppress("ConfidentialLedgerCertificateClient", typeof(Uri), typeof(TokenCredential))]
+    [CodeGenClient("ConfidentialLedgerCertificateClient")]
+    public partial class ConfidentialLedgerCertificateClient
     {
-        /// <summary> Initializes a new instance of ConfidentialLedgerIdentityServiceClient. </summary>
-        /// <param name="identityServiceUri"> The Identity Service URL, for example https://identity.accledger.azure.com. </param>
-        /// <exception cref="ArgumentNullException"> <paramref name="identityServiceUri"/> </exception>
-        public ConfidentialLedgerIdentityServiceClient(Uri identityServiceUri) : this(identityServiceUri, new ConfidentialLedgerClientOptions())
+        /// <summary> Initializes a new instance of ConfidentialLedgerCertificateClient. </summary>
+        /// <param name="certificateEndpoint"> The Identity Service URL, for example https://identity.accledger.azure.com. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="certificateEndpoint"/> </exception>
+        public ConfidentialLedgerCertificateClient(Uri certificateEndpoint) : this(certificateEndpoint, new ConfidentialLedgerCertificateClientOptions())
         {
         }
 
-        /// <summary> Initializes a new instance of ConfidentialLedgerIdentityServiceClient. </summary>
-        /// <param name="identityServiceUri"> The Identity Service URL, for example https://identity.accledger.azure.com. </param>
+        /// <summary> Initializes a new instance of ConfidentialLedgerCertificateClient. </summary>
+        /// <param name="certificateEndpoint"> The Identity Service URL, for example https://identity.accledger.azure.com. </param>
         /// <param name="options"> The options for configuring the client. </param>
-        /// <exception cref="ArgumentNullException"> <paramref name="identityServiceUri"/> </exception>
-        public ConfidentialLedgerIdentityServiceClient(Uri identityServiceUri, ConfidentialLedgerClientOptions options)
+        /// <exception cref="ArgumentNullException"> <paramref name="certificateEndpoint"/> </exception>
+        public ConfidentialLedgerCertificateClient(Uri certificateEndpoint, ConfidentialLedgerCertificateClientOptions options)
         {
-            if (identityServiceUri == null)
+            if (certificateEndpoint == null)
```

Comment:
Any reason that we're not using the Azure.Core `Argument` assertions?

Reply:
nope :) Will add.